### PR TITLE
[zh] update using-api/deprecation-guide.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] update using-api/deprecation-guide.md

### CODE REVIEW

**Comment by tengqm** [comment]
This file is out of sync. Please sync first.
/hold

File: content/zh/docs/reference/using-api/deprecation-guide.md [hunk 1]
```diff
@@ -741,25 +741,25 @@ to locate use of deprecated APIs.
...
 <!--
-Note that this may use non-ideal default values. To learn more about a specific
-resource, check the Kubernetes [API reference](/docs/reference/kubernetes-api/).
+  Note that this may use non-ideal default values. To learn more about a specific
+  resource, check the Kubernetes [API reference](/docs/reference/kubernetes-api/).
 -->
-注意这种操作生成的结果中可能使用的默认值并不理想。
-要进一步了解某个特定资源，可查阅 Kubernetes [API 参考](/zh/docs/reference/kubernetes-api/)。
+  注意这种操作生成的结果中可能使用的默认值并不理想。
+  要进一步了解某个特定资源，可查阅 Kubernetes [API 参考](/zh/docs/reference/kubernetes-api/)。
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/deprecation-guide.md:764]: whoops, these and the other leading spaces should be dropped.

Comment (tengqm) [reply content/zh/docs/reference/using-api/deprecation-guide.md:764]: Looks like the upstream English version has dropped these leading spaces.

**Comment by my-git9** [comment]
/

**Pull request closed without merging**